src/docker_clojure/dockerfile/lein.clj

```diff
@@ -54,6 +54,11 @@
           "mv lein-pkg $LEIN_INSTALL/lein"
           "chmod 0755 $LEIN_INSTALL/lein"
           "wget -q https://github.com/technomancy/leiningen/releases/download/$LEIN_VERSION/leiningen-$LEIN_VERSION-standalone.zip"
+          "wget -q https://github.com/technomancy/leiningen/releases/download/$LEIN_VERSION/leiningen-$LEIN_VERSION-standalone.zip.asc"
+          "gpg --batch --auto-key-locate keyserver --keyserver keys.openpgp.org --locate-keys phil@hagelb.org"
```

Comment:
This should still be pulling by-fingerprint so the key itself gets verified too (`20242BACBBE95ADA22D0AFD7808A33D379C806C3`), but otherwise this looks good! :metal:

```suggestion
          "gpg --batch --keyserver keys.openpgp.org --recv-keys 20242BACBBE95ADA22D0AFD7808A33D379C806C3"
```

Reply:
Oh I didn't notice I could commit that directly in here. Nifty! Thanks @tianon.